sympy/physics/control/tests/test_lti.py

```diff
@@ -239,7 +268,7 @@ def test_TransferFunction_functions():
     assert expect4_.evalf() == expect4
 
     # evaluate the transfer function at particular frequencies.
-    assert tf1.eval_frequency(wn) == wn**2/(wn**2 + 4*wn - 5) + 2*wn/(wn**2 + 4*wn - 5) - 3/(wn**2 + 4*wn - 5)
+    assert tf1.eval_frequency(wn) == wn/(wn + 5) + 3/(wn + 5)
```

Comment:
Rather than changing tests, it would be better to test symbolically like so:

```python
assert sm.simplify(tf1.eval_frequency(wn) - (wn**2/(wn**2 + 4*wn - 5) + 2*wn/(wn**2 + 4*wn - 5) - 3/(wn**2 + 4*wn - 5)))
```